common/config/azure-pipelines/templates/core-build.yaml

```diff
@@ -86,6 +86,12 @@ steps:
       CI: true
     condition: and(succeeded(), eq(variables['Agent.OS'], 'Windows_NT'))
 
+  # Run codemod tests
+  - script: npm run test
+    displayName: run Codemod Tests
+    workingDirectory: "tools/codemod"
+    condition: and(succeeded(), ne(variables['Agent.OS'], 'Linux'))
```

Comment:
Why is this condition needed?

Reply:
Due to unexpected failures only in Linux environment

Reply:
Ok let's keep this for now, need to determine why `jscodeshift` extension methods https://github.com/facebook/jscodeshift?tab=readme-ov-file#extensibility are not working on Linux.